src/main.js

```diff
@@ -277,13 +279,79 @@ let load = loader({
     },
   },
 
+  s3Controller: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
+      return new remoteS3.Controller({
+        region: cfg.app.blobArtifactRegion,
+        accessKeyId: cfg.aws.accessKeyId,
+        secretAccessKey: cfg.aws.secretAccessKey,
+      });
+    },
+  },
+
+  // This is a dependency to ease the creation of buckets for, e.g. testing
+  createBuckets: {
+    requires: ['cfg'],
+    setup: async ({cfg}) => {
```

Comment:
I think it's really nice to have it here because it means that instead of a big script, it's 10 lines of code here.  We're also going to use it so infrequently, I think it's not a huge concern.